### PR TITLE
[BB] Add bb integration page

### PR DESCRIPTION
**Pull request by sorta**
For https://travisci.assembla.com/spaces/Web/tickets/266/details

### CODE REVIEW

File: ember-cli-build.js [hunk 1]
```diff
@@ -62,6 +62,11 @@ module.exports = function () {
           { removeViewBox: false },
           { removeTitle: false },
           { removeDesc: false },
+          {
+            removeUnknownsAndDefaults: {
+              unknownContent: false,
```
Comment (sorta) [inline ember-cli-build.js]: svgo was removing drop shadows from one of the new SVGs. This change prevents that. Not 100% sure it's worth it though

**Comment by Nat-ess** [comment]
Hi @sorta, this is looking so good! Below are some small edits:
Edits

**Comment by sorta** [comment]
Thanks for the feedback @Nat-ess! I've made those changes, let me know if it needs anything else :smile:

**Comment by Nat-ess** [comment]
Looks good to me, thanks @sorta !

**Comment by piotr-travisci** [comment]
Thank you @sorta for an amazing job!
One question: What should happen when I sign up with BB account? Should all buttons "Sign up with Bitbucket" disappear from BB Integration page?

**Comment by sorta** [comment]
@piotr-travisci That's a very good question! It would make sense, though the page might look a little weird without them. Another option could be to redirect from the page if someone is logged in with bitbucket, though maybe that's weird too. WDYT @artursmirnov @Nat-ess @AlonaHlobina @pauladamgordon ?

**Comment by Nat-ess** [comment]
To be honest, I think this section should talk/tease users about Bitbucket integration and lead them to the Bitbucket landing page we've been working on–not a sign-up button.

File: app/templates/components/ui-kit/image.hbs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{{#if this.svg}}
+  <UiKit::Box
+    @display={{this.display}}
+    @margin={{this.margin}}
+    @height={{this.height}}
+    @width={{this.width}}
+  >
+    <SvgImage
+      @name={{this.svg}}
+      @alt={{this.alt}}
```
Comment (artursmirnov) [inline app/templates/components/ui-kit/image.hbs]: If these are just arguments that passthrough to the child component, then documentation recommends the following syntax:
```suggestion
      @name={{@svg}}
      @alt={{@alt}}
```
I'm not sure if it has any differences, but at least visually it indicates that those are external attributes.

File: public/images/svg/travis-console.svg [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+<path class="st15" d="M70.274,270.892v-0.005c-4.078-0.466-7.309-3.704-7.766-7.785h-0.004c-0.002-0.03-0.026-0.055-0.057-0.055
+	c-0.031,0-0.055,0.025-0.057,0.055h-0.003c-0.458,4.085-3.694,7.326-7.779,7.787v0.003c-0.03,0.002-0.055,0.026-0.055,0.057
+	c0,0.031,0.025,0.055,0.055,0.057v0.003c4.08,0.46,7.315,3.696,7.777,7.775h0.006c0.002,0.03,0.026,0.055,0.057,0.055
+	c0.031,0,0.055-0.025,0.057-0.055h0.004c0.462-4.075,3.69-7.308,7.765-7.773v-0.005c0.03-0.002,0.055-0.026,0.055-0.057
+	C70.329,270.918,70.304,270.894,70.274,270.892z"/>
+<g>
+	
+		<ellipse transform="matrix(0.149 -0.9888 0.9888 0.149 157.2976 416.1463)" class="st16" cx="320.416" cy="116.689" rx="5.547" ry="5.547"/>
+</g>
+</svg>
```
Comment (artursmirnov) [inline public/images/svg/travis-console.svg]: This image looks a lot like a "Mastercard" to me 😄

File: app/components/ui-kit/text.js [hunk 4]
```diff
@@ -68,6 +76,7 @@ const SIZES = {
   XL2: '2xl',
   XL3: '3xl',
   XL4: '4xl',
+  XL4_5: '4.5xl',
```
[the remark this answers is not in the document]
Comment (sorta) [reply app/components/ui-kit/text.js]: Good question. For some reason I thought so at some point, there's a decent size difference between `XL4` and `XL5`. But I tried `XL4` just now on the relevant items, and I think it actually looks better. So I think switching to `XL4` and getting rid of 4.5 sounds good to me! Change made

File: app/routes/integration.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+export default BasicRoute.extend(TailwindBaseMixin, {
+  needsAuth: false,
+
+  features: service(),
+
+  redirect() {
+    if (!this.get('features.proVersion')) {
+      return this.transitionTo('/');
+    }
+  }
```
[the remark this answers is not in the document]
Comment (sorta) [reply app/routes/integration.js]: Ahh that makes sense! Change made, thanks for the info :smile:

File: app/routes/integration/index.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import BasicRoute from 'travis/routes/basic';
+
+export default BasicRoute.extend({
+  redirect() {
```
[the remark this answers is not in the document]
Comment (sorta) [reply app/routes/integration/index.js]: Change made :)

File: public/images/svg/travis-console.svg (hunk 3 above)
Comment (sorta) [reply public/images/svg/travis-console.svg]: Ohhhh yeah, it totally does XD haha

**Comment by sorta** [comment]
Thanks for the review @artursmirnov! I think I've addressed all the current feedback at this point. I'm gonna give folks a little time for final review, and unless something comes up this should be ready to merge into the epic branch tomorrow. Thanks everyone!

**Pull request merged**